packaging/vm-deployment/provision.yaml

```diff
@@ -0,0 +1,90 @@
+---
+- hosts: all
+  become: yes
+  become_method: sudo
+  strategy: debug
+  tasks:
+
+  - name: Set Variables
+    set_fact:
+      vagrant_user: "vagrant"
+
+  - name: Update OS
+    package:
+      name: '*'
+      state: latest
+
+  - name: Enable IPv6
+    command: sysctl -w net.ipv6.conf.all.disable_ipv6=0
+
+  - name: check if cargo is installed
+    become: no
+    shell: command -v cargo
+    register: cargo_exists
+    ignore_errors: yes
+
+  - name: Download Rust Installer
```

Comment:
It would probably be better to build the binaries outside of the test machine IMHO.